index.js

```diff
@@ -0,0 +1,51 @@
+let books = [];
+let i = 0;
+
+function BookList(b) {
+  return `
+    <li>${b.title}</li>
+    <li>${b.author}</li>
+    <button type='button' id='${b.id}' class='remove-button'>Remove</button>`;
+}
+
+const removeBook = (ev) => {
+  const buttonId = ev.target.id;
+  books = books.filter((y) => y !== books[books.findIndex((x) => x.id === parseInt(buttonId, 10))]);
+  localStorage.setItem('bookObject', JSON.stringify(books));
+  document.getElementById('Result').innerHTML = `${books.map(BookList).join('')}`;
+};
+// Add Form data to Unordered List
+const addBooks = (ev) => {
+  ev.preventDefault();
+  i += 1;
+  const book = {
+    id: i,
```

Comment:
* I think you should make your id unique currently when the remove button is clicked on identical id numbers both are removed. please kindly fix this 🙏 as it deletes the wrong book

Reply:
I fixed it

Reply:
nice 👍